notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>##Supported APIs  -&gt; ## Supported APIs </p><p> </p><p> </p><p>Right now we support 4 different model types in our APIs/SDK -&gt;</p><p> </p><p>Currently, 4 model types are supported in the APIs/SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Fixed.</p>